scripts/wine-pyinstaller.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/usr/bin/env bash -e
```

Comment:
FWIW, this doesn't work on Linux: https://stackoverflow.com/questions/4303128/how-to-use-multiple-arguments-for-awk-with-a-shebang-i-e